src/game/GameStateMatch.cpp

```diff
@@ -45,11 +45,14 @@ bool GameStateMatch::load() {
     //gives the player control of the marine
     player.setControl(GameManager::instance()->getMarine(playerMarineID));
     player.getMarine()->setPosition(newPoint.first, newPoint.second);
-
+    player.getMarine()->setSrcRect(0, 0, 75, 125);
```

Comment:
Im not a fan of magic numbers,
you can use `static constexpr int NAME = value;`/ Check out Player.h for an example.